ChangeLog

```diff
@@ -103,24 +103,8 @@ Changes
    * Close a test gap in (D)TLS between the client side and the server side:
      test the handling of large packets and small packets on the client side
      in the same way as on the server side.
-   * Change the dtls_client and dtls_server samples to work by default over
-     IPv6 and optionally by a build option over IPv4.
-   * Change the use of Windows threading to use Microsoft Visual C++ runtime
-     calls, rather than Win32 API calls directly. This is necessary to avoid
-     conflict with C runtime usage. Found and fixed by irwir.
-   * Remember the string format of X.509 DN attributes when replicating
-     X.509 DNs. Previously, DN attributes were always written in their default
-     string format (mostly PrintableString), which could lead to CRTs being
-     created which used PrintableStrings in the issuer field even though the
-     signing CA used UTF8Strings in its subject field; while X.509 compliant,
-     such CRTs were rejected in some applications, e.g. some versions of
-     Firefox, curl and GnuTLS. Reported in #1033 by Moschn. Fix contributed by
-     Thomas-Dee.
-   * Improve documentation of mbedtls_ssl_get_verify_result().
-     Fixes #517 reported by github-monoculture.
-   * Add MBEDTLS_MPI_GEN_PRIME_FLAG_LOW_ERR flag to mbedtls_mpi_gen_prime() and
-     use it to reduce error probability in RSA key generation to levels mandated
-     by FIPS-186-4.
+   * Reduce the stack consumption of mbedtls_x509write_csr_der() which previously
```

Comment:
This seems to be a rebase artefact: The ChangeLog entry should go at the top of the ChangeLog, preserving all existing entries. Also, since no ChangeLog affecting PRs have been merged since 2.14, we need to crate a new temporary section `= mbed TLS x.xx.x branch released xxxx-xx-xx` to hold the entries for the next release.

Reply:
Sorry for that. fixed it!